tests/components/mqtt/test_light.py

```diff
@@ -383,12 +383,17 @@ async def test_invalid_state_via_topic(hass, mqtt_mock, caplog):
     assert not state.attributes.get(ATTR_ASSUMED_STATE)
 
     async_fire_mqtt_message(hass, "test_light_rgb/status", "1")
+    async_fire_mqtt_message(hass, "test_light_rgb/rgb/status", "255,255,255")
+    async_fire_mqtt_message(hass, "test_light_rgb/brightness/status", "255")
+    async_fire_mqtt_message(hass, "test_light_rgb/color_temp/status", "153")
+    async_fire_mqtt_message(hass, "test_light_rgb/effect/status", "none")
+    async_fire_mqtt_message(hass, "test_light_rgb/white_value/status", "255")
 
     state = hass.states.get("light.test")
     assert state.state == STATE_ON
     assert state.attributes.get("rgb_color") == (255, 255, 255)
     assert state.attributes.get("brightness") == 255
-    assert state.attributes.get("color_temp") == 150
+    assert state.attributes.get("color_temp") == 153
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)